horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/SchemaServiceImpl.java

```diff
@@ -272,13 +270,19 @@ public void dropToken(int id) {
    }
 
    public String updateToken(int id, String token) {
-      Query query = em.createNativeQuery(UPDATE_TOKEN);
```

Comment:
I am +100 for removing native queries, we lose type safety in the code and couple ourselves to the db schema with every native query we have, esp for simple cases like this

Reply:
Good to hear that, I think there are many other places (in other services) where we could remove them. I will try do that as part of other testing improvements :rocket: